lms/templates/view.html

```diff
@@ -28,23 +29,39 @@ <h1>{{ _('תצוגת תרגיל') }} {{ solution['exercise']['id'] }}: {{ soluti
       {% endfor -%}
       </ul>
     </nav>
-    {% if is_manager %}
+    {% if config.SHAREABLE_SOLUTIONS %}
+    <div class={{ direction }}>
+      <strong>{{ _('שיתוף תרגיל') }}:</strong>
+    </div>
+    <div id="share-solution" class="share-function">
+      <span id="solution-link" title="Share solution"><i class="fa fa-share-alt"></i></span>
+      <div id="share-content-box">
+        <div id="share-box-input">
+          <span id="cancel-share" title="Disable share solution"><i class="fa fa-trash"></i></span>
+          <span id="copy-link" title="Copy shared link"><i class="fa fa-copy"></i></span>
+          <input type="text" id="link-input" readonly>
+        </div>
+      </div>
+    </div>
+    {% endif -%}
+    {% if is_manager and not is_shared %}
```

Comment:
Why not is_shared?

Reply:
Because it's only code-view screen. All the users will see the code only